sdk/README.md

```diff
@@ -12,142 +12,149 @@
-		process.exit(1);
-	});
+Execute the `lisk init` command of Lisk Commander as shown below, to bootstrap a default blockchain client in the current folder.
+
+```sh
+lisk init
 ```
 
-After that you can start the application by:
+As a result of executing the `init` command, you will be asked for the _Name_, _ChainID_, _Description_, _Author_, and _License_ of the new blockchain client.
 
 ```
-node index.js
+Using template "lisk-ts"
+Initializing git repository
+Updating .liskrc.json file
+Creating project structure
+? Application name my_client
+? Chain ID in hex representation. ChainID must be 4 bytes (8 characters) 12345678
+? Application description A simple blockchain client for my Web3 application
+? Author XYZ
+? License ISC
+```
+
+Next, all the required files are created by Lisk Commander.
+
+It is now already possible to start the newly bootstrapped blockchain client with default configurations for a local Devnet:
+
+```sh
+./bin/run start
 ```
 
 ### Configure your blockchain parameters
 
-You can also define your blockchain application parameters such as `blockTime`, `maxTransactionsSize` and more with an optional configurations object.
-
-```js
-const { app } = Application.defaultApplication(genesisBlockDevnet, {
-    genesisConfig: {
-      blockTime: 5,
-      maxTransactionsSize: 100 * 1024,
-      minRemainingBalance: "5000000",
-	  activeValidators: 101,
-	  standbyValidators: 2,
-	  validatorListRoundOffset: 2
-    },
-    ...
-});
+The blockchain client offers a variety of configuration options such as `blockTime`, `maxTransactionsSize` and more.
+
+For a complete overview of the client configuration options, please check out the [SDK configuration reference](https://lisk.com/documentation/lisk-sdk/v6/config.html) and the [Blockchain client configuration](https://liskhq.github.io/lisk-docs/beta/build-blockchain/configuration.html) guide.
+
+To use the custom config with the client, use the `--config` flag.
+Also add the `--overwrite-config` flag, if you used another config the last time the client was started.
+
+```sh
+./bin/run start --config config/custom_config.json --overwrite-config
 ```
 
 ### Register a custom module or a custom plugin
 
-A custom module is a logic to define state changes that will be executed on-chain meaning that it will be a part of the blockchain protocol.
-On the other hand, a custom plugin is a logic to define an off-chain logic which is not part of the blockchain protocol but to enhance the application features.
+A custom [module](https://lisk.com/documentation/beta/understand-blockchain/sdk/modules-commands.html) is a logic to define state changes that will be executed on-chain meaning that it will be a part of the blockchain protocol.
+
+A custom [plugin](https://lisk.com/documentation/beta/understand-blockchain/sdk/plugins.html), on the other hand, is a logic to define an off-chain logic which is not part of the blockchain protocol but to enhance the blockchain application features.
```

Comment:
```suggestion
A custom [module](https://lisk.com/documentation/beta/understand-blockchain/sdk/modules-commands.html) defines the logic that makes state changes on-chain, meaning that it will be a part of the blockchain protocol.

A custom [plugin](https://lisk.com/documentation/beta/understand-blockchain/sdk/plugins.html), on the other hand, defines an off-chain logic that is not part of the blockchain protocol but enhances the blockchain application features.
```